### PR TITLE
Update offset values and add util functions

### PR DESCRIPTION
**Pull request by TanBeige**

### CODE REVIEW

File: .DS_Store
Comment (papag00se) [inline .DS_Store:1]: Let's add .DS_Store to the .gitignore file

File: src/utils/getMaxOffset.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+export const getMaxOffset = (zoom: number, borderExists: boolean = false) => {
+    const maxOffset: number = ((zoom || 100) / 100 - 1) * (borderExists ? 576 : 636)
```
Comment (papag00se) [inline src/utils/getMaxOffset.ts]: Good change.

We decided to only go with the 576 number as we'd have to duplicate this in the contract and it is not as straightforward there. 576 pixels of movement is plenty even if the border doesn't exist.

**Review by papag00se — DISMISSED** [review]
Looks good! Just a couple of small changes

File: src/utils/getMaxOffset.ts (hunk 1 above)
Comment (TanBeige) [reply src/utils/getMaxOffset.ts]: 576 is plenty, makes it cleaner to use on the frontend as well. Fixed

**Review by bigirishlion — APPROVED** [review]
🥍

**Review by papag00se — APPROVED** [review]
LGTM

**Pull request merged**